features/evaluation.feature

```diff
@@ -1,5 +1,8 @@
 Feature: Flag evaluation
 
+  Background:
+    Given an openfeature client is registered with cache disabled
+
```

Comment:
Is this necessary to mention?

Reply:
Do you mean is it necessary to mention at all or is it necessary to include `with cache disabled`?

Reply:
I'm specifically wondering about `with cache disabled`. This is the "vanilla" test suite, I'd like to not pollute it with any concepts like caching.

I guess you're worried that enabling caching here will cause unexpected behavior in the test suite?

Reply:
Exactly that, as caching is enabled by default this causes unexpected behaviour, I thought it best to explicitly disable it.